client/verta/verta/client.py

```
@@ -946,8 +947,13 @@ def log_code(self, exec_path=None, repo_url=None, commit_hash=None, overwrite=Fa
             # write ZIP archive
             zipstream = six.BytesIO()
             with zipfile.ZipFile(zipstream, 'w') as zipf:
-                # TODO: save notebook
-                zipf.write(exec_path, os.path.basename(exec_path))  # write as base filename
+                filename = os.path.basename(exec_path)
+                try:
+                    saved_notebook = _utils.save_notebook(exec_path)
+                except:  # not a notebook, or failed to save
```

I'm not sure we should catch all and fail silently if it failed to save. That's not what I would expect as a user. Maybe at least print a warning?